simapp/simd/cmd/root.go

```diff
@@ -235,8 +236,9 @@ func queryCommand() *cobra.Command {
 	cmd.AddCommand(
 		authcmd.GetAccountCmd(),
 		rpc.ValidatorCommand(),
-		rpc.BlockCommand(),
```

Comment:
I think this is a wrapper of the tendermint command, while the new one is sdk esk version of it. Is the result the same?

Reply:
Yeah, the resulting info is the same - except we now can query by hash as well as block with it.